lib/src/bbox.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:turf/helpers.dart';
+import 'package:turf/meta.dart';
+
+/// Calculates the bounding box for any [geoJson] object, including [FeatureCollection].
+/// Uses geojson.[bbox] if available and options.[recompute] is not set.
```

Comment:
I think you can remove the `options.` and `geojson.`, otherwise great

Reply:
Resolved in #96